build/uploadperftobenchview.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+# The intent of this script is upload produced performance results to BenchView in a CI context.
+#    There is no support for running this script in a dev environment.
+
+if ! env | grep -q ^perfWorkingDirectory=
```

Comment:
I think a better way to do this may be something like this:

    if [[ -z $perfWorkingDirectory ]]; then

Reply:
@dsplaisted 
We are checking to see if the value is set - I can add the check for existence.

Reply:
done...